docs/references/kontena-yml.md

```diff
@@ -331,26 +366,12 @@ links:
   - mysql:wordpress-mysql
```

Comment:
Hmm... AFAIK the link aliases don't actually work yet, in terms of actually generating DNS entries? Should we be documenting them yet?